scripts/aws/cfn/microservice-dev-cfg.json

```diff
@@ -6,9 +6,9 @@
     "TokenExchangeLambdaEnvironmentTokenTtl": "3200",
     "TokenExchangeLambdaEnvironmentIssuer":"https://webapi.dev.pn.pagopa.it",
     "TokenExchangeLambdaEnvironmentAllowedIssuer":"https://hub-login.ecs.dev.pn.pagopa.it,https://hub-login.spid.dev.pn.pagopa.it,https://spid-hub-test.dev.pn.pagopa.it,https://dev.selfcare.pagopa.it,https://uat.selfcare.pagopa.it",
-    "TokenExchangeLambdaEnvironmentAllowedOrigin":"https://portale-pa.dev.pn.pagopa.it,https://portale.dev.pn.pagopa.it,https://portale-pa-develop.fe.dev.pn.pagopa.it",
+    "TokenExchangeLambdaEnvironmentAllowedOrigin":"https://portale-pa.dev.pn.pagopa.it,https://portale.dev.pn.pagopa.it,https://portale-pg.dev.pn.pagopa.it,https://portale-pa-develop.fe.dev.pn.pagopa.it,https://portale-pg-develop.fe.dev.pn.pagopa.it",
```

Comment:
Chiedi a Carlotta se esistono questi 

- https://portale-pa-develop.fe.dev.pn.pagopa.it
- https://portale-pg-develop.fe.dev.pn.pagopa.it

Reply:
I record non esistono più, li ho eliminati dalla variabile `TokenExchangeLambdaEnvironmentAllowedOrigin`